Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -2433,7 +2433,9 @@ private func _XCTExpectFailure(
   strict: Bool = true,
   failingBlock: () -> Void
 ) {
+
   #if DEBUG
+  #if os(macOS)
```

Comment:
Is this ours or upstream? If ours, why `macOS` and not `Darwin`? When do we decide?